commerce/bookstore/python/load_data.py

```diff
@@ -0,0 +1,286 @@
+import csv, uuid, random
+from typedb.client import TypeDB, SessionType, TransactionType
+
+data_path = "data/"  # path to csv files to import/load data
+db = 'bookstore'  # Name of the DB to connect on the TypeDB
+
+
+def parse_data_to_dictionaries(input):
+    """
+      1. reads the file through a stream,
+      2. adds the dictionary to the list of items
+      :param input.file as string: the path to the data file, minus the format
+      :returns items as list of dictionaries: each item representing a data item from the file at input.file
+    """
+    #print('Parsing of ' + input["file"] + 'started.')
+    items = []
+
+    with open(input["file"] + ".csv", encoding='UTF-8') as data:  # 1
+        for row in csv.DictReader(data, delimiter=";", skipinitialspace=True):
+            item = {key: value for key, value in row.items()}  # fieldnames (keys) are taken from the first row
+            items.append(item)  # 2
+    print('Parsing of ' + input["file"] + ' successful.')
+    return items
+
+
+def load_data_into_typedb(input, session):
+    """
+      loads the csv data into our TypeDB phone_calls database:
+      1. gets the data items as a list of dictionaries
+      2. for each item dictionary
+        a. creates a TypeDB transaction
+        b. constructs the corresponding TypeQL insert query
+        c. runs the query
+        d. commits the transaction
+      :param input as dictionary: contains details required to parse the data
+      :param session: off of which a transaction will be created
+    """
+    items = parse_data_to_dictionaries(input)  # 1
+    for item in items:  # 2
+        with session.transaction(TransactionType.WRITE) as transaction:  # a
+            TypeQL_insert_query = input["template"](item)  # b # This calls one of the _template functions
```

Comment:
Let's use consistent casing. It makes a substantial difference to readability; a name starting with an uppercase letter looks like a class/enum, while a name starting with a lowercase letter looks like a variable or parameter.

```suggestion
            typeql_insert_query = input["template"](item)  # b # This calls one of the _template functions
```
Apply this change to all Python scripts. There may be a way to do this automatically in one click using IntelliJ or PyCharm's refactoring tools.

Reply:
I have fixed that at https://github.com/vaticle/typedb-examples/pull/143/commits/bac11c1f22a9e32656a3a486261a39c8a21646c0